log/log.go

```diff
@@ -149,13 +150,52 @@ type Logger interface {
 	Fatalln(...interface{})
 	Fatalf(string, ...interface{})
 
+	Log(keyvals ...interface{}) error
+
 	With(key string, value interface{}) Logger
 }
 
 type logger struct {
 	entry *logrus.Entry
 }
 
```

Comment:
Missing doc string, should explain the behavior, the expectations of the input interface and what errors will be raised.